### ISSUE TITLE
Override AnimationOptions on the page/component

### ISSUE THREAD

**Issue by DataTowerSystems** [aboudoux/BlazorAnimation#10]
Hello,
Thank you very much for sharing this greate tools.
Is there any way to override Animation Options on the page/component, because I want most of my pages have the same effects so I made a default options in the program.cs ;
builder.Services.Configure<AnimationOptions>(o =>
```
{
    o.Effect = BlazorAnimation.Effect.FadeInDown;
    o.Speed = Speed.Slow;
    o.Delay = Delay.None;
    o.IterationCount = 1;
}); 
```
But some time I want to change the effects of the animation for the element, Can I override the configured options?
Again thank you so much for your efforts and time.

**Comment by Arturace** [comment]
I think the [Named Configurations](https://github.com/aboudoux/BlazorAnimation#named-configurations) are made for this use case.

If you use the default configuration, the [ApplyConfigurations](https://github.com/aboudoux/BlazorAnimation/blob/6f5a15f39a506b8ae395579466899d4a43b47c97/BlazorAnimation/Animation.razor#L71) will always grab the default configuration property instead of the component's parameter, unless that default configuration property is not set.

I would suggest flipping this behaviour: taking the default options value if the parameter is not set.

**Comment by Arturace** [comment]
As a workaround, you might be able to give an empty string of a non-existent configuration name to the `OptionsName` parameter.

**Comment by aboudoux** [comment]
Hi, The update will be available in version 2.3.0
